src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header id="home" class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__link">Apple</a>
```

Comment:
```suggestion
            <a href="#" class="is-active nav__link">
              Apple
            </a>
```

Such formatting will improve code readability
![261037525-ac21c9ae-e3c6-4961-ba19-d74dcf11db92](https://github.com/mate-academy/layout_moyo-header/assets/75898310/9965f304-e3a2-4fb8-9f73-1cb2fb1c5a6b)